Gruntfile.js

```diff
@@ -314,6 +314,25 @@ module.exports = function(grunt) {
         files: {
           'test/actual/pages_metadata/': ['test/fixtures/pages/blog/index.hbs']
         }
+      },
+      // Using glob pattern for layouts
+      glob_layout: {
+        options: {
+          layouts: 'test/fixtures/layouts/**/*.hbs',
+          layoutdir: undefined
```

Comment:
it seems like it would be better to make `layoutdir` act like a `cwd` for layouts, so that, if defined, the glob pattern passed to `layouts` would use that as a base. so you could do:

```js
layouts: '**/*.hbs',
layoutdir: 'test/fixtures/layouts/'
```

at least I was thinking that's what you were going to do. thoughts?
sorry for taking so long! I was on vacation for a few days.